openpower/package/hostboot/p9Patches/hostboot-0003-remove-unsed-var-attn-rt.patch

```diff
@@ -0,0 +1,26 @@
+From 8e9ebe09e1656fab860bb0d5bf1da2f6aad9378d Mon Sep 17 00:00:00 2001
+From: Matt Ploetz <maploetz@us.ibm.com>
+Date: Tue, 9 Aug 2016 10:21:39 -0500
+Subject: [PATCH] Remove unused var
+
+Change-Id: I563e862d078cb99ac0083b58b80608acfd431597
+---
+ src/usr/diag/attn/runtime/attn_rt.C | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/usr/diag/attn/runtime/attn_rt.C b/src/usr/diag/attn/runtime/attn_rt.C
+index 62860d9..0675174 100644
+--- a/src/usr/diag/attn/runtime/attn_rt.C
++++ b/src/usr/diag/attn/runtime/attn_rt.C
+@@ -105,7 +105,7 @@ namespace ATTN_RT
+         int rc = 0;
+         errlHndl_t err = NULL;
+         AttentionList attentions;
+-        MemOps & memOps = getMemOps();
++        //MemOps & memOps = getMemOps();
```

Comment:
Another issue open for this one?  Or is it being handled on an internal bug tracker?